### ISSUE TITLE
[BUG] Accessibility/VoiceOver: Buttons at the top of your own profile page not announced properly

### ISSUE THREAD

**Issue by ericyahhh** [mastodon/mastodon-ios#543]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

- Go to your own profile by tapping on the far right button in the tab bar
- Turn on VoiceOver
- Focus the buttons at the top

Button announcements include the icon shapes:
- "Star, Button, Star"
- "Square and arrow, Button, Share"
- "Cog, Button, Settings"

### Expected Behavior

Button should be announced – similar to the settings button on the front page – as following:
- "Star profile, Button"
- "Share profile, Button"
- "Settings, Button"

### Steps To Reproduce

See above

### Environment

```markdown
- Device: iPhone 12 mini
- OS: iOS 16.1
- Version: 1.4.5
- Build: 144
```

**Comment by j-f1** [comment]
Thanks for reporting! Opened #544 to fix.
